EstateAdministrationUI.Testing/TestData.cs

```diff
@@ -0,0 +1,303 @@
+﻿namespace EstateAdministrationUI.Testing
+{
+    using System;
+    using System.Collections.Generic;
+    using Areas.Estate.Models;
+    using BusinessLogic.Models;
+    using EstateManagement.DataTransferObjects.Responses;
+
+    public class TestData
+    {
+        public static Guid EstateId = Guid.Parse("FC657462-2620-4B35-B744-1BD7E60C1748");
+
+        public static String EstateName = "Test Estate 1";
+
+        public static String OperatorName = "Test Operator 1";
+
+        public static Guid OperatorId = Guid.Parse("DECA8293-F045-41C5-A2F7-30F2792FD273");
+
+        public static Boolean RequireCustomMerchantNumber = true;
+
+        public static Boolean RequireCustomTerminalNumber = true;
+
+        public static String EmailAddress = "estateuser1@testestate1.co.uk";
+
+        public static Guid SecurityUserId = Guid.Parse("6A3C1B74-F01E-4017-85D5-6E6082038AB8");
+
+        public static String MerchantAddressLine1 = "Address Line 1";
+
+        public static String MerchantAddressLine2 = "Address Line 2";
+
+        public static String MerchantAddressLine3 = "Address Line 3";
+
+        public static String MerchantAddressLine4 = "Address Line 4";
+
+        public static String MerchantContactEmailAddress = "testcontact@merchant1.co.uk";
+
+        public static String MerchantContactName = "Mr Test Contact";
+
+        public static String MerchantContactPhoneNumber = "1234567890";
+
+        public static String MerchantCountry = "United Kingdom";
+
+        public static String MerchantName = "Test Merchant 1";
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5756318)